ios/history.md

```diff
@@ -5,6 +5,7 @@
 * Bookmark add button is enabled only when title/url fields have text (#1073)
 * Removed deprecated code to to support keyman:// scheme for ad-hoc distribution (#1160)
 * Updated the default keyboard to SIL EuroLatin (#1101)
+* Fixed bug behind some crashes of system keyboard (#1166)
```

Comment:
FYI, while you were gone, @mcdurdin went with the direction of history.md files referring to PR numbers (in the past, we used issue number)